src/utils/notificationsLocalStorage.ts

```diff
@@ -73,10 +73,10 @@ export const setProcessSuccess = () => {
   );
 };
 
-export const setProcessError = (error: unknown) => {
+export const setProcessError = (error: any) => {
   const accountIdLS = getLSAccountId();
   const localStorageByAccountId = getNotificationLocalStorage();
-
+  const errorMessage = error.message ? error.message : "";
```

Comment:
@milenakrawczyk perhaps we should record `"unknown"` instead of `""` as this could be understood as 'no error occurred'

Reply:
Yes, `''` was confusing. I fixed it.